src/daemon/rpmostreed-os-experimental.cxx

```diff
@@ -193,7 +194,8 @@ osexperimental_handle_download_packages (RPMOSTreeOSExperimental *interface,
   OstreeRepo *repo = ostree_sysroot_repo (sysroot);
   g_autoptr(RpmOstreeContext) ctx = rpmostree_context_new_client (repo);
   rpmostree_context_set_dnf_caching (ctx, RPMOSTREE_CONTEXT_DNF_CACHE_FOREVER);
-
+  g_autofree char * source_name;
```

Comment:
Need to initialize this to `NULL`, i.e. `g_autofree char * source_name = NULL;`.